crates/proof-of-sql/src/sql/proof_plans/empty_exec.rs

```diff
@@ -67,8 +67,8 @@ impl ProverEvaluate for EmptyExec {
         &self,
         _alloc: &'a Bump,
         _accessor: &'a dyn DataAccessor<S>,
-    ) -> Vec<Column<'a, S>> {
-        Vec::new()
+    ) -> Table<'a, S> {
+        Table::<'a, S>::try_new(IndexMap::default()).unwrap()
```

Comment:
This is where it make sense to set a default number of rows, since this is where that decision gets made.